src/interfaces.ts

```diff
@@ -98,17 +104,17 @@ export enum SyncStatus {
 }
 
 export interface ConnectionHandler {
-    name: string
+    id: string
     enabled: boolean
-    config: Record<string, string>
+    config: BaseHandlerConfig
 }
 
 export interface Connection {
     _id?: string
     _rev?: string
     name: string
```

Comment:
There's a remaining `name` to replace by `id`

Reply:
Sorry, I don't follow this one.

Reply:
In this interface, we should rename `name` by `id` to be consistent with the rest.

The result of `GET /connections` and other routes return this objects following this interface.

Reply:
This is an object that is stored in the Verida database, hence it uses `_id`, instead of `id` and has a `name` property that is standard across all Verida datastore objects.

Reply:
Yeah ok, but as a consumer using this Data Connections API we don't deal with Verida records directly as there are dedicated endpoints, so having to use `_id` or `name` is inconsistent with the other `id` used in the other objects.

Also, we see again the ambiguity of `name` because in the Verida records it's usually a label/title displayed in the UI of the Wallet or in the Data tab of the Vault web app. (for a data connection, it would be the same as `readableId`)

Anyway, I won't be using `name`, I'll use `_id`